lib/fragmentSender.js

```diff
@@ -80,6 +81,7 @@ module.exports = function(chat, fragments, payload, initialMessage = '', initial
                 timing: push.timing,
                 pushId: push.id,
                 reportId: payload.report,
+                reportIndex: nextReportIndex - 1,
```

Comment:
Simplify by not adding `1` when determing index and not subtracting `1` when reporting